### DISCUSSION TITLE
Appimage flags

### DISCUSSION THREAD

**Discussion by raisedintheusa** [Alex313031/thorium#41] [Q&A] [answered]
Hi, is there a way to change the command-line flags on the Appimage? Like if I want to remove the --autoplay-policy flag, is there a way to do that? Thanks.

**Comment by Alex313031** [comment, marked as answer]
@raisedintheusa  What you can do is build the appimage yourself. Just download the thorium repo, as well as the latest .deb file.
Just use this directory > https://github.com/Alex313031/Thorium/tree/main/infra/APPIMAGE and you can just drop the latest .deb in that folder and modify the *Thorium.yml* file, then run `build_appimage.sh` The Thorium.yml file actually writes the internal thorium-browser.desktop file, so you can edit all the `Exec` lines and add/remove flags as needed. You can ignore the thordeb.sh part because that part is only for contributors who are building the full browser. build_appimage.sh should only take 1-2 minutes.
